cmdeploy/src/cmdeploy/cmdeploy.py

```diff
@@ -331,19 +341,23 @@ def get_parser():
     return parser
 
 
+def get_sshexec(ssh_host: str, verbose=True):
+    if ssh_host in ["localhost", "@local"]:
+        return "localhost"
+    elif ssh_host == "@docker":
+        return "@docker"
+    if verbose:
+        print(f"[ssh] login to {ssh_host}")
+    return SSHExec(ssh_host, verbose=verbose)
```

Comment:
this looks weird to return strings or an object instance.  Usually functions should return a common type. 
Instead i would return a LocalExec instance or DockerExec instance, next to the SSHExec instance. 
roughly like 

```
class LocalExec:
     def __init__(self, verbose=False):
          self.verbose = verbose 
     def logged(self, func, **kwargs):
           if self.verbose:
               print(f"calling locally: {func.__name__}(**{kwargs})")
           return func(**kwargs)

```
and DockerExec similar.  So get_ssh_exec would return something where you can call "logged()" on.